Assignment2/assignment2.sql

```diff
@@ -1,40 +1,38 @@
 create index idx_batter_counts on baseball.batter_counts(game_id);
 create index idx_game on baseball.game(game_id);
 
+create table baseball.results (batter numeric, annual_batting_average double PRECISION, historic_batting_average double precision, rolling_batting_average double PRECISION);
 
 Question 1
 
 annual
```

Comment:
make sure to comment out these labels for the code because it won't run if you don't

Reply:
Hi tom, I changed the code a little bit and created the tables at the beginning of the queries, so I commented out this one since I no longer need it, can you check it one more time?

Reply:
So I commented out that table that I initially created, and stored all results in their own tables (baseball. annual_batting_average, baseball.total_batting_average, baseball.rolling_batting_average). this should work now.

Reply:
Also bit make sure to drop your temporary tables

Reply:
Bita*